packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -16,7 +16,7 @@ import { __ } from '@wordpress/i18n';
  */
 import BlockIcon from '../block-icon';
 
-export default function BlockNavigationList( {
+export default function __experimentalBlockNavigationList( {
```

Comment:
Do you think we can keep the name as is internally and just rename the one exported in the public API?

Reply:
@youknowriad Yep, that's now been changed 👍